packages/page-staking/src/Suspensions/Suspensions.tsx

```diff
@@ -92,6 +85,16 @@ function parseEvents (events: EventRecord[], productionBanConfigPeriod: number,
     }).flat();
 }
 
+function updateLiftEraForBansLiftedManually (eventsInBlocks: SuspensionEvent[], banned: Banned[], sessionInfo: SessionInfo) {
+  eventsInBlocks.forEach((eventInBlock) => {
+    if (banned.find((banned) => banned.account === eventInBlock.address) === undefined) {
+      if (eventInBlock.suspensionLiftsInEra > sessionInfo.currentEra) {
+        eventInBlock.suspensionLiftsInEra = sessionInfo.currentEra;
+      }
```

Comment:
We don't know when exactly a lift happened, so we show the `currentEra`, is that correct? Could we show nothing? Perhaps that would look like incomplete data and users would think they are still banned? WDYT?

Reply:
You're right that showing here always current era is not correct, as you noticed that e.g. ban was from 1018 till 1028, and lift happened in 1020, and then this ban will always be shown as lifted in the current era. We can show here something like "unbanned on request" or any other string, but it is too mouthful I think. The most important thing is to let user know this ban is no longer in place.